components/Map/Map.js

```diff
@@ -87,19 +86,20 @@ let events = [
       showsUserLocation={true}
       style={styles.map}
       region={{
-        latitude: 39.781511,
-        longitude: -104.798673,
-        latitudeDelta: 0.05,
-        longitudeDelta: 0.05
+        latitude: 39.7306645,
+        longitude: -104.988245,
+        latitudeDelta: 0.07,
+        longitudeDelta: 0.07
```

Comment:
I like having map default to viewing all of Denver proper. Curious when this will come into play with a user's current location, or if we'll event see that while we're in development?